packages/core/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -61,6 +78,37 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
         return Promise.resolve();
     }
 
+    private async _handleOnCancel(_data: OnCancelData) {
+        const lineItems = this._getLineItems();
+        const existingConsignment = this._store.getState().consignments.getConsignmentsOrThrow();
```

Comment:
We should rename this variable, because it contains an array of consignments:

```suggestion
        const existingConsignments = this._store.getState().consignments.getConsignmentsOrThrow();
```